utils/common/src/main/java/org/apache/brooklyn/util/javalang/coerce/TypeCoercerExtensible.java

```diff
@@ -165,13 +165,6 @@ public static TypeCoercerExtensible newEmpty() {
             }
         }
         
-        //for enums call valueOf with the string representation of the value
-        if (targetType.isEnum()) {
-            result = EnumTypeCoercions.tryCoerceUntyped(Strings.toString(value), (Class<T>)targetType);
-            if (result!=null && result.isPresent()) return result;
-            if (result!=null && firstError==null) firstError = result;
-        }
-
         //now look in registry
         synchronized (registry) {
```

Comment:
Interesting! Conceptually it could, but I think I'll leave as-is because the code using this wants to call `TypeCoercerExtensible.registerAdapter()` so it needs to know about the `registry`.